packages/nx-aws-cache/src/tasks-runner/aws-cache.ts

```diff
@@ -246,7 +246,7 @@ export class AwsCache implements RemoteCache {
       await this.s3.send(params);
       return true;
     } catch (err) {
-      if ((err as Error).name === 'NotFound') {
+      if ((err as Error).name === '403' || (err as Error).name === 'NotFound') {
```

Comment:
Is this correct, that the name equals status code? Should it be `Forbidden` or not?

Reply:
@leighpascoe